src/containers/LocationPage/LocationPage.jsx

```diff
@@ -33,6 +34,8 @@ import './LocationPage.scss';
 // Define how to read/write state to URL query parameters
 const urlQueryConfig = {
   viewMetric: { type: 'string', defaultValue: 'download', urlKey: 'metric' },
+  compareMetricFirst: { type: 'string', defaultValue: 'download', urlKey: 'first' },
+  compareMetricLast: { type: 'string', defaultValue: 'upload', urlKey: 'last' },
```

Comment:
Can we rename these urlKeys to something else instead of `first` and `last`? Maybe `compareX` and `compareY` ?